### PR TITLE
Enable authselect features clobbered by FreeIPA install scripts

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR enables several features in the active `authselect` profile so that all necessary hardened rules will be activated.

## 💭 Motivation and Context
This will help our AMI be more secure as well as pass several audits in a compliance scan.

## 🧪 Testing
All automated tests pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
LGTM!

File: files/00_setup_freeipa.sh [hunk 1]
```diff
@@ -155,6 +155,14 @@ function setup {
     ipa host-add-principal \
         "$hostname" \
         host/"$(curl --silent http://169.254.169.254/latest/meta-data/instance-id)"."$domain"
+
+    # Enable features in the active authselect profile so that all necessary
+    # hardened rules will be activated.  Note that these features are enabled
+    # in ansible-role-hardening, however ipa-server-install and
+    # ipa-client-install clobber them, so we must re-enable them here.
```
Comment (jsf9k) [inline files/00_setup_freeipa.sh]: I suggest adding to the comment a sentence addressing the fact that these commands are RedHat-only, but so are IPA servers.

**Review by mcdonnnj — APPROVED** [review]
Makes sense. LGTM ✔

File: files/00_setup_freeipa.sh (hunk 1 above)
Comment (dav3r) [reply files/00_setup_freeipa.sh]: Thanks - see updated comment in https://github.com/cisagov/ansible-role-freeipa-server/pull/23/commits/1b0fb61563810eee0a3ce3bed5c12c75ca95aa2e.

**Review by jsf9k — APPROVED** [review]
Approved for a second time!

**Pull request merged**